src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.Configurer
+{
+    public class DotnetFirstTimeUseConfigurer
+    {
+        private INuGetCachePrimer _nugetCachePrimer;
+        private INuGetCacheSentinel _nugetCacheSentinel;
+
+        public DotnetFirstTimeUseConfigurer(INuGetCachePrimer nugetCachePrimer, INuGetCacheSentinel nugetCacheSentinel)
+        {
+            _nugetCachePrimer = nugetCachePrimer;
+            _nugetCacheSentinel = nugetCacheSentinel;
+        }
+
+        public void Configure()
+        {
+            if(ShouldPrimeNugetCache())
+            {
+                PrintFirstTimeUseNotice();
+
+                _nugetCachePrimer.PrimeCache();
+            }
+        }
+
+        private void PrintFirstTimeUseNotice()
+        {
+            Reporter.Output.WriteLine();
+            Reporter.Output.WriteLine("Welcome to .NET Core!");
```

Comment:
I wasn't sure if here it was going to have fidelity to how I wanted it printed. I saw that in other places we did it like this, so I assumed not.

I will change.